Header header
schunk_sdh_ros/ContactInfo[] contact_info